Libraries/StyleSheet/processTransform.js

```diff
@@ -115,9 +115,15 @@ function _convertToRadians(value: string): number {
   return value.indexOf('rad') > -1 ? floatValue : floatValue * Math.PI / 180;
 }
 
-function _validateTransforms(transform: Object): void {
+function _validateTransforms(transform: Array<Object>): void {
```

Comment:
The prop in style is called transform so I guess this is fine, I just fixed the flow type as it was bad, flow didn't catch the error but it always was an array that is passed to this function.